crates/sui-core/src/epoch/randomness.rs

```diff
@@ -741,8 +632,18 @@ impl RandomnessManager {
         &mut self,
         batch: &mut DBBatch,
         authority: &AuthorityName,
-        conf: VersionedDkgConfimation,
+        conf: VersionedDkgConfirmation,
     ) -> SuiResult {
+        // confirmation was received from other validators, so we need to ensure it uses a supported
+        // version before we call other functions that assume the version is correct
+        let dkg_version = self.epoch_store()?.protocol_config().dkg_version();
+        if !conf.is_valid_version(dkg_version) {
+            warn!(
+                "ignoring DKG Confirmation from authority {authority:?} with unsupported version"
```

Comment:
is this safe or could it result in a fork?

Reply:
Assuming the version is the same on all honest validators, all validators will ignore this message.